app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -5,6 +5,7 @@
   id ||= "default-id-#{SecureRandom.hex(4)}"
   items ||= []
   anchor_navigation ||= false
+  event_tracking ||= false
```

Comment:
Would a better name for this variable be something like `track_show_all_clicks`?

Reply:
Works for me, will update